cmd/ps3netsrv-go/handler.go

```diff
@@ -51,7 +51,20 @@ func (h *Handler) HandleReadDir(ctx *server.Context) []os.FileInfo {
 		return []os.FileInfo{}
 	}
 
-	return entries
+	// Resolve symbolic links
+	var files []os.FileInfo
+	for _, entry := range entries {
+		// Stat the entry to get its file info
+		info, err := h.Fs.Stat(*ctx.Cwd + "/" + entry.Name())
```

Comment:
Same notes as before:
1. Please use `filepath.Join`
2. Probably we can do less system calls here: `Stat` don't needed if file type is not symlink. For symlinks `info.Mode() & os.ModeSymlink` should be non-zero.

Reply:
Fixed